components/Careers/ApplyNewJob.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+<div class="font-nunito text-base text-black dark:text-white dark:bg-slate-900">
+    <!-- Start Hero -->
+    <section class="relative table w-full py-36 lg:py-44 bg-[url('../../assets/images/job/job.jpg')] bg-no-repeat bg-center bg-cover">
+        <div class="absolute inset-0 bg-black opacity-80"></div>
+        <div class="container">
+            <div class="grid grid-cols-1 pb-8 text-center mt-12">
+                <h3 class="mb-4 md:text-4xl text-3xl md:leading-normal leading-normal font-medium text-white">Senior Web Developer</h3>
+
+                <ul class="list-none">
+                    <li class="inline text-slate-400 ltr:mr-3 rtl:ml-3"><i class="uil uil-map-marker text-white h6 ltr:mr-1 rtl:ml-1"></i> London, UK - <span class="text-white">Full Time</span></li>
+                </ul>
+            </div>
+            <!--end grid-->
+        </div>
+        <!--end container-->
+    </section>
+    <!--end section-->
+    <div class="relative">
+        <div class="shape absolute right-0 sm:-bottom-px -bottom-[2px] left-0 overflow-hidden text-white dark:text-slate-900">
+            <svg class="w-full h-auto" viewBox="0 0 2880 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M0 48H1437.5H2880V0H2160C1442.5 52 720 0 720 0H0V48Z" fill="currentColor"></path>
+            </svg>
+        </div>
+    </div>
+    <!-- End Hero -->
+
+    <!-- Start Section-->
+    <section class="relative md:py-24 py-16">
+        <div class="container">
+            <div class="grid lg:grid-cols-12 grid-cols-1" id="reserve-form">
+                <div class="lg:col-start-2 lg:col-span-10">
+                    <div class="rounded-md shadow dark:shadow-gray-800 bg-white dark:bg-slate-900 p-6">
+                        <form>
+                            <div class="grid lg:grid-cols-12 gap-6">
+                                <div class="lg:col-span-6">
+                                    <div class="ltr:text-left rtl:text-right">
+                                        <FormField label="Name" help="Required. Your name" :error="errorMsgs.name">
+                                            <FormControl
+                                              v-model="formData.name"
+                                              name="username"
+                                              required
+                                              autocomplete="username"
+                                            />
+                                          </FormField>                                    </div>
```

Comment:
Please do proper code formatting, can you use esLint extension

Reply:
have a look